### DISCUSSION TITLE
Improve icons for assistants to reflect their apprenticeship

### DISCUSSION THREAD

**Discussion by Arctomachine** [FakeFishGames/Barotrauma#11957] [Ideas and suggestions]
Right now it is not always clear which job an assistant belongs to from just quickly looking at assigned orders in crew list. And even less clear when looking at character lists in other places.
A corresponding character icon after picking apprenticeship can help with this.

1. Half/half. Take left half of assistant and right half of apprenticeship job. Maybe not exactly half and allow overlaps to cover empty space (which security and mechanic have in my example)
![image](https://user-images.githubusercontent.com/29041820/236433659-be4fcaf0-ca9c-480d-acb9-d6f0670eff25.png)

2. Small indicator. Take small icon of apprenticeship and put it in a corner of assistant
![image](https://user-images.githubusercontent.com/29041820/236436045-214a925c-19e7-4956-9b10-c96d0add08de.png)

3. Reversed indicator. Same as above, but reversed. Icon is replaced with apprenticeship job and small assistant is added in a corner

4. Just color. Recolor whole icon into color of apprenticeship. It is bad indication, so this option should not be used as primary solution

**Comment by BiffW** [comment]
Great idea! I would prefer version 2.

**Comment by MarkCringan** [comment]
I know when hiring crew assistants their icons tend to reflect what they have the most skill in.

I think the best option is to just make the class icons white. 

Example: specializing in security will make the icon look like a security icon but white instead of brown.
